ethereum/forge-test/Bridge.t.sol

```diff
@@ -56,6 +146,1289 @@ contract TestBridge is Test {
         // evmChainId must equal block.chainid
         vm.expectRevert("invalid evmChainId");
         bridge.setEvmChainIdPub(1337);
+    }
+
+    function testShouldBeInitializedWithTheCorrectSignersAndValues() public {
```

Comment:
Not sure this should be a test. We could move these assertions to the `setup` function.

Reply:
Agree this should be addressed with #4090